Sources/StreamChat/Workers/ChannelUpdater.swift

```diff
@@ -325,23 +325,59 @@ class ChannelUpdater: Worker {
 
     /// Add users to the channel as members.
     /// - Parameters:
+    ///   - currentUserId: the id of the current user.
     ///   - cid: The Id of the channel where you want to add the users.
-    ///   - users: User Ids to add to the channel.
+    ///   - userIds: User ids to add to the channel.
+    ///   - message: Optional system message sent when adding a member.
+    ///   - messageId: Optional id of the message.
     ///   - hideHistory: Hide the history of the channel to the added member.
     ///   - completion: Called when the API call is finished. Called with `Error` if the remote update fails.
-    func addMembers(cid: ChannelId, userIds: Set<UserId>, hideHistory: Bool, completion: ((Error?) -> Void)? = nil) {
-        apiClient.request(endpoint: .addMembers(cid: cid, userIds: userIds, hideHistory: hideHistory)) {
+    func addMembers(
+        currentUserId: UserId? = nil,
+        cid: ChannelId,
+        userIds: Set<UserId>,
+        message: String? = nil,
+        messageId: String? = nil,
```

Comment:
Why is message id passed here if we are never using it?

Reply:
We're using it only for the tests, to compare the endpoint

Reply:
Yes, I agree, found a better way.